src/analyses/ai.cpp

```diff
@@ -289,27 +294,36 @@ bool ai_baset::visit(
 
 bool ai_baset::visit_edge(
   const irep_idt &function_id,
-  locationt l,
+  trace_ptrt p,
```

Comment:
I'd appreciate `p` being named something more informative (just "trace_ptr"?) here and elsewhere. The various "l_" things at least referred to "location" rather than just "pointer" (to what?)

Reply:
Fair point.  As there is already going to be a PR for source code text changes that don't alter the functionality, can I address this as part of that?

Reply:
I'm OK with that approach, but it might be helpful to open an issue to track all the name-related cleanup that needs to be done so that things like this can be logged and then checked in the PR for text-only changes later - otherwise I fear they might get forgotten...